regions/shapes/range.py

```diff
@@ -0,0 +1,72 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import Angle, SkyCoord
+from regions.core.attributes import QuantityLength
+from ..core import SkyRegion, PixelRegion, PixCoord
+
+__all__ = ["RangePixelRegion", "RangeSphericalRegion"]
+
+inf = Angle(float("inf"), "deg")
+
+
+# TODO: needed?
```

Comment:
I don't think this is needed.  Isn't it covered exactly by the existing planar rectangle as long as you choose the right parameterization?  It might be thought of as a "convenience function".  Maybe an additional constructor / factory function for the planar rectangle?

Reply:
I think there's a few small advantages to introducing`RangePixelRegion` (instead of using RectanglePixelRegion):

- Clear code, symmetry, being able to round-trip when doing `to_pixel` and `from_pixel` isntead of mixing over to the rectangle classes
- I'm not sure how the +- inf case would be handled. I don't think one can handle the `min=-inf, max=42` case, there's no center that will result in a rectangle extending that range?
- It's not a lot of work (30 lines of code?)

I'm still not sure if it's worth adding, but my feeling is that it might be the best way to go to add it?

@gpdf - Thoughts?

If no addition, what exactly whould be the alternative? 
Should `RangeSphericalRegion` not offer `to_pixel` at all, or should it go to `RectanglePixelRegion`?